pic-sure-hpds-ui/src/main/webapp/picsureui/common/router.js

```diff
@@ -225,11 +225,19 @@ define([
 
                 filterList.init(settings.picSureResourceId, outputPanelView, JSON.parse(parsedSess.queryTemplate));
             },
-            displayGoogleAnalytics: function () {
+            displayGoogleAnalytics: function() {
                 let analyticsView = new googleAnalytics.View({analyticsId: settings.analyticsId});
-                // append the analytics view to the body
                 analyticsView.render();
                 $("head").append(analyticsView.$el);
+                $("body").prepend(
+                '<!-- Google Tag Manager (noscript) --> \
```

Comment:
This might add slashes visually? I dunno. Try shutting off your scripts and see if this displays right.

Reply:
Good point about the noscript. moved.